test/e2e/swaps/shared.js

```diff
@@ -160,11 +160,11 @@ const checkNotification = async (driver, options) => {
     '[data-testid="swaps-banner-title"]',
   );
   assert.equal(await boxTitle.getText(), options.title, 'Invalid box title');
-  const boxContent = await driver.findElement(
-    '[data-testid="mm-banner-alert-notification-text"]',
-  );
+  const boxContent = await driver.findElement({
+    css: '[data-testid="mm-banner-alert-notification-text"]',
+    text: options.text,
+  });
   const bodyText = await boxContent.getText();
```

Comment:
Lines 167 to 173 become unnecessary after the change you made above

Reply:
Good point. I had first written it that way because our desire is to still be able to write e2e tests by making assertions, and having assertion style error outputs when tests fail.

To address your comment I made commit 1843e1a874, which removes some of the redundancy while still allowing the desired assertions syntax.

There is more discussion of this here https://docs.google.com/document/d/1eQyeScSZQlzGLapef_8PANkU_ObNPJcuMCP040T2gfM/edit#heading=h.3jasxj89sekd